### ISSUE TITLE
[i18n] Requesting Polish config string improvements.

### ISSUE THREAD

**Issue by YukisCoffee** [Rehike/Rehike#205]
@PANMARSAG14 Sorry to make this request, but I made some accessibility changes to the configuration page strings and would like to mirror that on the Polish i18n. The new strings should be more user-friendly overall.

**Comment by PANMARSAG14** [comment]
Okay, i finished improving Polish config strings.

**Comment by aubymori** [comment]
Merged.
